radar/2023-09-12/github-actions.md

```diff
@@ -5,4 +5,4 @@ quadrant:   tools
 tags:       [infrastructure]
 ---
 
-Used extensively for CI across CHT products including testing and releasing.
\ No newline at end of file
+[GitHub Actions](https://github.com/features/actions) is used extensively for Continuous Integration (CI) across CHT products including testing and releasing.
```

Comment:
Do you think we should add "GitHub" as well?

Reply:
Great idea - done!